spring-security-modules/spring-security-mvc-boot-1/src/main/java/org/baeldung/custom/config/SecurityConfig.java

```diff
@@ -2,12 +2,25 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
 @Configuration
-public class SecurityConfig {
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
+    @Override
+    protected void configure(final HttpSecurity http) throws Exception {
+        http.csrf()
```

Comment:
Can we format this new block properly using the appropriate formatter e.g eclipse:

- https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml

Reply:
Done. Sorry about this omission.